### ISSUE TITLE
Stuck Initializing Download

### ISSUE THREAD

**Issue by MuggleStruggle** [RiskiVR/BSLegacyLauncher#27]
```
[10:25:42.725][Assembly-CSharp][Info] - Triggered login again?
[10:25:42.770][Assembly-CSharp][Info] - Connecting to Steam3...
[10:25:43.571][Assembly-CSharp][Info] -  Done!
[10:25:43.572][Assembly-CSharp][Info] - Logging 'superduperfakeusername' into Steam3...
[10:25:43.653][Assembly-CSharp][Info] - Successfully connected to Steam3 service!
[10:25:44.219][Assembly-CSharp][Info] - Disconnected from Steam
[10:25:44.220][Assembly-CSharp][Info] - This account is protected by Steam Guard.
[10:25:44.220][Assembly-CSharp][Info] - Steamguard Triggered
[10:25:44.221][Assembly-CSharp][Debug] - SteamGuard Triggered
[10:25:44.221][Assembly-CSharp][Info] - Unable to get steam3 credentials.
[10:26:08.722][Assembly-CSharp][Info] - Triggered login again?
[10:26:08.724][Assembly-CSharp][Info] - Connecting to Steam3...
[10:26:09.159][Assembly-CSharp][Info] -  Done!
[10:26:09.160][Assembly-CSharp][Info] - Logging 'superduperfakeusername' into Steam3...
[10:26:09.161][Assembly-CSharp][Info] - Successfully connected to Steam3 service!
[10:26:09.559][Assembly-CSharp][Info] -  Done!
[10:26:09.560][Assembly-CSharp][Info] - Using Steam3 suggested CellID: 75
[10:26:09.561][Assembly-CSharp][Info] - You selected version 1.16.2:3692829915208062825
[10:26:09.591][Assembly-CSharp][Info] - Got 255 licenses for account!
[10:26:09.597][Assembly-CSharp][Info] - Got session token!
[10:26:10.113][Assembly-CSharp][Info] - Got AppInfo for 620980
[10:26:10.308][Assembly-CSharp][Info] - Steam account owns Beat Saber!
[10:26:10.309][Assembly-CSharp][Info] - Using app branch: 'public'.
[10:26:10.405][Assembly-CSharp][Info] - Accepted new login key for account superduperfakeusername
[10:26:10.656][Assembly-CSharp][Info] - Got depot key for 620981 result: OK
[10:26:10.661][Assembly-CSharp][Info] - Processing depot 620981 - Beat Saber Content
[10:26:10.667][Assembly-CSharp][Info] - Downloading depot manifest...
[10:26:11.108][Assembly-CSharp][Info] - Got CDN auth token for steamcontent.com result: Fail (expires 1/1/1970 12:00:00 AM)
[10:26:11.110][Assembly-CSharp][Info] - Disconnected from Steam

[10:35:50.155][Assembly-CSharp][Info] - Triggered login again?
[10:35:50.270][Assembly-CSharp][Info] - Connecting to Steam3...
[10:35:50.757][Assembly-CSharp][Info] -  Done!
[10:35:50.757][Assembly-CSharp][Info] - Logging 'superduperfakeusername' into Steam3...
[10:35:50.797][Assembly-CSharp][Info] - Successfully connected to Steam3 service!
[10:35:51.018][Assembly-CSharp][Info] -  Done!
[10:35:51.019][Assembly-CSharp][Info] - Using Steam3 suggested CellID: 75
[10:35:51.021][Assembly-CSharp][Info] - You selected version 1.16.2:3692829915208062825
[10:35:51.085][Assembly-CSharp][Info] - Got 255 licenses for account!
[10:35:51.091][Assembly-CSharp][Info] - Got session token!
[10:35:51.364][Assembly-CSharp][Info] - Got AppInfo for 620980
[10:35:51.562][Assembly-CSharp][Info] - Steam account owns Beat Saber!
[10:35:51.563][Assembly-CSharp][Info] - Using app branch: 'public'.
[10:35:51.755][Assembly-CSharp][Info] - Got depot key for 620981 result: OK
[10:35:51.761][Assembly-CSharp][Info] - Processing depot 620981 - Beat Saber Content
[10:35:51.765][Assembly-CSharp][Info] - Downloading depot manifest...
[10:35:51.954][Assembly-CSharp][Info] - Got CDN auth token for steamcontent.com result: Fail (expires 1/1/1970 12:00:00 AM)
[10:35:51.959][Assembly-CSharp][Info] - Disconnected from Steam``

Launcher gets stuck on initializing download. Nothing happens.  Left it idle for a while, nothing.  Ended up having to force close it.

**Comment by zb1ackcat** [comment]
Hello,

I am seeing a simular error with the 1970 error

[20:22:06.853][Assembly-CSharp][Info] - Triggered login again?
[20:22:06.942][Assembly-CSharp][Info] - Connecting to Steam3...
[20:22:07.336][Assembly-CSharp][Info] -  Done!
[20:22:07.337][Assembly-CSharp][Info] - Logging 'DJD' into Steam3...
[20:22:07.369][Assembly-CSharp][Info] - Successfully connected to Steam3 service!
[20:22:07.703][Assembly-CSharp][Info] -  Done!
[20:22:07.703][Assembly-CSharp][Info] - Using Steam3 suggested CellID: 83
[20:22:07.705][Assembly-CSharp][Info] - You selected version 1.18.0:5599254819160454367
[20:22:07.725][Assembly-CSharp][Info] - Got 75 licenses for account!
[20:22:07.729][Assembly-CSharp][Info] - Got session token!
[20:22:08.151][Assembly-CSharp][Info] - Got AppInfo for 620980
[20:22:08.401][Assembly-CSharp][Info] - Steam account owns Beat Saber!
[20:22:08.402][Assembly-CSharp][Info] - Using app branch: 'public'.
[20:22:08.641][Assembly-CSharp][Info] - Got depot key for 620981 result: OK
[20:22:08.644][Assembly-CSharp][Info] - Processing depot 620981 - Beat Saber Content
[20:22:08.647][Assembly-CSharp][Info] - Downloading depot manifest...
[20:22:08.896][Assembly-CSharp][Info] - Got CDN auth token for steamcontent.com result: Fail (expires 1/1/1970 12:00:00 AM)
[20:22:08.899][Assembly-CSharp][Info] - Disconnected from Steam

**Comment by zb1ackcat** [comment]
Acoording to this CDN is no  longer supported https://github.com/SteamRE/DepotDownloader/releases/tag/DepotDownloader_2.4.5

**Comment by RiskiVR** [comment]
Sorry for not responding. We're aware of the issue and will update DepotDownloader when we can

**Comment by zb1ackcat** [comment]
> Sorry for not responding. We're aware of the issue and will update DepotDownloader when we can

No need to appologize. Thank you so much for all your work.

**Comment by RiskiVR** [comment]
Fixed in v1.3.3
